.github/workflows/reuse.yaml

```diff
@@ -152,7 +135,7 @@ jobs:
           -a "repo=${{ github.repository }}" \
           -a "workflow=${{ github.workflow }}" \
           -a "ref=${{ github.sha }}" \
-          ghcr.io/${{ github.repository_owner }}/${{inputs.image_name}}@sha256:${{ steps.get-step-image.outputs.digest }}
+          ${{ steps.ko-build-dev.outputs.digest }}
```

Comment:
shouldn't it be "ko-build" instead of "ko-build-dev"?

Reply:
Close! The `ko build dev image` step should have had `id: ko-build-dev` so that the `Sign dev image` can use its output.

I'm surprised GitHub let me define two steps with the same `id` but 🤷 -- this just points out that we'll need to do more testing of this release workflow before we can merge this PR.